### ISSUE TITLE
L036 includes whitespace as a select target which should be on a new line

### ISSUE THREAD

**Issue by joellabes** [sqlfluff/sqlfluff#646]
In the sql 
```
combined as (
    select 
        renamed.*,
        coalesce(academic_years.is_currently_active, false) as is_current_academic_year
    from renamed
    left outer join academic_years using (academic_year_id)
)
```
, there is a space after `select`. This triggers two linting rules:
```
L:  29 | P:   5 | L036 | Select targets should be on a new line unless there is
                       | only one select target.
L:  29 | P:  11 | L001 | Unnecessary trailing whitespace.
```

Whitespace is covered by L001, so I'd say it shouldn't also trigger L036. 

`python --version`
Python 3.8.3 (Windows)

`sqlfluff --version`
sqlfluff, version 0.4.0a3

**Comment by barrywhart** [comment]
I have a PR #652 which I believe fixes the issue.
